<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherWidget</class>
 <widget class="QWidget" name="WeatherWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>161</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WeatherWidget</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="icon_temp_layout" stretch="0,0,1,0">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer name="spacer_left">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="icon_label">
       <property name="styleSheet">
        <string notr="true">* {
  font-family: qweather-icons;
  font-size: 24px;
}
.weather-today QLabel {
  color: #ee8545;
}</string>
       </property>
       <property name="text">
        <string></string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="temp_label">
       <property name="styleSheet">
        <string notr="true">* {
  font-family: Microsoft YaHei;
  font-size: 20px;
}
.weather-today QLabel {
  font-weight: bold;
  color: #ee8545;
}</string>
       </property>
       <property name="text">
        <string>10℃~20℃</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer_right">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="date_weekday_label">
     <property name="styleSheet">
      <string notr="true">* {
  font-weight: 300;
  font-family: Microsoft YaHei;
  font-size: 19px;
}
.weather-today QLabel{
  font-weight: bold;
  color: #ee8545;
}</string>
     </property>
     <property name="text">
      <string>1月1日 星期一</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
